service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/SendDocumentController.java

```diff
@@ -14,18 +14,24 @@
 import uk.gov.hmcts.reform.ccd.client.model.CallbackRequest;
 import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
 import uk.gov.hmcts.reform.fpl.model.CaseData;
-import uk.gov.hmcts.reform.fpl.model.Representative;
+import uk.gov.hmcts.reform.fpl.model.Recipient;
 import uk.gov.hmcts.reform.fpl.model.SentDocument;
 import uk.gov.hmcts.reform.fpl.model.SentDocuments;
 import uk.gov.hmcts.reform.fpl.model.common.DocumentReference;
 import uk.gov.hmcts.reform.fpl.model.common.Element;
-import uk.gov.hmcts.reform.fpl.service.DocumentSenderService;
+import uk.gov.hmcts.reform.fpl.service.SendLetterService;
 import uk.gov.hmcts.reform.fpl.service.SentDocumentHistoryService;
 
+import java.util.ArrayList;
 import java.util.List;
 
 import static uk.gov.hmcts.reform.fpl.enums.RepresentativeServingPreferences.POST;
 
+/*
+@deprecated - use SendDocumentService instead
+*/
+
+@Deprecated
```

Comment:
why deprecate instead of remove? just curious

Reply:
ah I see, existing places have not been changed for now 👍 e.g CMO, NoH

Reply:
should we make tech debt ticket to update existing code?

Reply:
will do cleanup at the end of series of stories